libs/mark-flow-ui/src/pages/contest_page.tsx

```diff
@@ -92,15 +103,60 @@ export function ContestPage(props: ContestPageProps): JSX.Element {
     }
   })();
 
+  const nextContestButtonRef = useRef<Button>(null);
+  const handleOnPressNext = useCallback(() => {
+    const to = nextContest
+      ? getContestUrl(nextContestIndex)
+      : getReviewPageUrl();
+    history.push(to);
+    // Need to blur or the focus on "Next" button will be retained for next contest,
+    // increasing chance the voter will accidentally skip a contest
+    nextContestButtonRef?.current?.blur();
+  }, [getContestUrl, getReviewPageUrl, history, nextContest, nextContestIndex]);
+
   const nextContestButton = (
-    <LinkButton
+    <Button
```

Comment:
Converted to `Button` so we can call `focus()` and `blur()` on it. Consequently needed to reimplement `to` behavior. Checked with @kofi-q and this is aligned with our direction for `LinkButton`, which we want to reserve for true web browser contexts only.

Reply:
Without code changes, the only way that might focus `LinkButton` out of the box is to pass the `autoFocus` prop. I found that doing that adds highlight styles but doesn't focus it ie. sending the PAT equivalent of "Enter" doesn't actually navigate to the next contest.

If we wanted to make changes to `LinkButton` we could use refs like `Button` does to handle focus and blur, or convert `LinkButton` to a class component so we can expose methods that wrap `Button.focus()` and `Button.blur()`. So the question is whether we want to invest in `LinkButton`, which is related to your second question.

Based on my conversation with @kofi-q, my understanding is `LinkButton` main benefit is when the user is in a regular web browser and we want to support actions like using the back button or `ctrl + click` for a new tab. Since we're not using a regular browser for the machine apps the preferred direction is to move away from `LinkButton` and back to `Button`. Therefore, while we could build out focus/blur handling in `LinkButton`, it's preferable to use `Button` which already has that handling.

Reply:
Thanks for sharing all of that context.

Since `LinkButton` is implemented with `Button` under the hood, I think it should be as simple as forwarding the ref to the `Button` to gain support for `focus` and `blur`. IMO, that's basically equivalent to what you've done here, which reimplements the same logic as inside `LinkButton` (and I'm pretty sure still supports the back button implicitly by way of the `history` object). I think there may be a slight misunderstanding since our `LinkButton` isn't using an `a` tag under the hood, which would support `ctrl+click` and such.

Reply:
Reverted back to LinkButton and used `forwardRef` in [5b74493](https://github.com/votingworks/vxsuite/pull/4284/commits/5b74493d7373f84c16c864de5a9319cb3a51bdaf).

Back button was my own assumption but you're right it would be handled by `history` anyway.